doc/sphinx/source/figuresofmerit/index.rst

```diff
@@ -0,0 +1,184 @@
+Chi square figures of merit
+================================================================================
+
+Within the NNPDF methodology various figures of merit are used, each of which
+can be used in different situations. To avoid confusion, it is important to
+understand the differences between the various figures of merit, and to
+understand which definition we are referring to in a given context. In
+particular, it is worth stressing that whenever a figure of merit is discussed,
+the :math:`t_0` method (discussed below) applies.
+
+Here we we provide an overview of the different figures of merit, and discuss
+when each of them is used.
+
+
+The basis of the loss functions: 𝜒²
```

Comment:
I feel that here it is somewhat ambiguous that we are discussing the so called "experimental" chi2, and seems to me the word "experimental" should appear in a few places.

Reply:
So this is something that confused me a bit about the discussion for the referee reply, perhaps you can clear this up. What we call the "experimental" chi2 in n3fit uses the same t0 covmats as the validation and training losses, however based on the mail exchange this week it seems that the "experimental" chi2 doesn't use the t0 prescription?

Reply:
Perhaps I can chime in here. For all our fits we use the covariance matrix constructed under the t0 prescription. That is: multiplicative uncertainties are multiplied by predictions from the t0 PDF (as opposed to the experimental measurement) to transform them to additive uncertainties before constructive the covariance matrix.

The reason we do this has to do with the d'agostini bias: if we attempt to use the non-t0 covmat (which is the experimental covmat) to compute the loss, then our results will be a biased estimator.

The discussion for the reply to the referee pertains to what covmat we use to compute the chi2 tables in the reports. The experimental chi2 uses the experimental covmat; the t0 chi2 similarly uses the t0 covmat. Really we should look at the t0 chi2 because that is what the replicas are optimising for.

Reply:
Thanks, but this is indeed my point: what you now call the "t0 chi2" is simply called "experimental chi2" (or sometimes "true chi2" iirc) inside n3fit. 

For external communications I suppose that's fine, as long as we're consistent in distinguishing t0 chi2 from experimental chi2, but in the docs this double use of the name can cause problems/confusion.